django_fido/views.py

```diff
@@ -230,6 +230,9 @@ def complete_registration(self, form: Form) -> AuthenticatorData:
         except UnsupportedType as error:
             _LOGGER.info("FIDO 2 registration failed with error: %r", error)
             raise ValidationError(_('Security key is not supported because it cannot be identified.'), code='invalid')
+        except InvalidAttestation as error:
+            _LOGGER.info("FIDO2 registration failed with error: %r", error)
+            raise ValidationError(_('Registration failed.'), code='invalid')
```

Comment:
What about designing more specific error message? Registration failed is not very helpful. We could at least suggest that there is a problem with security key attestation.

Reply:
This can really mean quite a lot of things. Bad signature, incorrect certificate used, bad format of attestation data and maybe something else. We should probably stay general, so what about:
`Registration failed, incorrect data from token.`?

Reply:
Sounds good.